Makefile

```diff
@@ -91,7 +91,7 @@ deploy-db-migration: ## Deploys the db migration app
 	$(if ${APPLICATION_NAME},,$(error Must specify APPLICATION_NAME))
 	cf push ${APPLICATION_NAME}-db-migration -f <(make -s -C ${CURDIR} generate-manifest) -o digitalmarketplace/${APPLICATION_NAME}:${RELEASE_NAME} --no-route --health-check-type none -i 1 -m 128M -c 'sleep 2h'
 	cf stop ${APPLICATION_NAME}-db-migration
-	cf run-task ${APPLICATION_NAME}-db-migration "cd /app && python application.py db upgrade" --name ${APPLICATION_NAME}-db-migration
+	cf run-task ${APPLICATION_NAME}-db-migration "cd /app && venv/bin/python application.py db upgrade" --name ${APPLICATION_NAME}-db-migration
```

Comment:
`VIRTUALENV_ROOT` ?

Reply:
That won't be defined at this point, I think, because the venv hasn't been activated. That var gets set once you activate it.

Reply:
Maybe not?

Reply:
Can we call the `virtualenv` step at the beginning of this step (as per some of the other steps, e.g. `make generate-manifest`)?